lib/bulk_data/cache.rb

```diff
@@ -44,8 +44,7 @@ def initialize
       @cache = ActiveSupport::Cache::RedisCacheStore.new(
         namespace: "content-publisher:bulk-data-cache-#{Rails.env}",
         error_handler: ->(exception:, **) { GovukError.notify(exception) },
-        reconnect_attempts: 3,
-        reconnect_delay: 0.1,
+        reconnect_attempts: [0.1, 0.2, 0.3],
```

Comment:
Should this be `[0.1, 0.1, 0.1]` to match previous behaviour? Reading: [https://github.com/dentarg/redis-rb/blob/371d8c1effced9e1e65a2ffcd0ed01a89c5c2573/README.md#reconnections sounds](https://github.com/redis/redis-rb#reconnections) like the wait is after a fail, but I'm struggling to determine it authoritatively.

Reply:
Ah i think you're right 
`# retry a third and final time after another 1s` 

Apologies for the poor reading comp.